explainers/wpub-explainer.md

```diff
@@ -107,18 +107,24 @@ Note how we used `rel=contents` to identify that a particular resource is an HTM
 ## Design choices
 
 
-The key questions are [1] identifying the "bounds" of the publication, [2] defining the ordering of the primary resources, and [3] figuring out how to express metadata for the publication as a whole. 
+The design of web publications has caused perhaps the greatest amount of discussion within the group, and centers around three key areas:
 
+1. how best to identify the "bounds" of the publication;
+2. how best to define the ordering of the primary resources; and
+3. how best to express metadata for the publication as a whole.
 
-### 1. What's part of the publication?
+This section outlines the current approaches that have been adopted in each of these areas, and lists alternative practices that have been discussed and/or influenced the decision-making process.
 
-- The [Web Application Manifest spec](https://w3c.github.io/manifest/) uses `scope` to define the extent of a web application. 
 
-- Every component of an EPUB publication must be listed in a `manifest` element in an XML package file.
+### 1. What is part of the publication?
 
-- Web sites do not explicitly define their boundaries.
+The web publications working group has chosen to use an explicit list of resources in the manifest to identify the bounds, similar to how EPUB approaches the problem. Anything that is not part of the `resources` or `readingOrder` manifest members is considered to be outside the web publication. The group has also considered the patterns described below, but currently believes an explicit list is the simpler choice.
```

Comment:
Maybe it is worth saying that the scope or transclusion approaches (but I think we also considered the usage of URI patterns, b.t.w.) may come back for later versions, and the decision for the simpler choice was also a pragmatic to ensure an easy transition for the publ. community.